Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/ObjCStubs.kt

```diff
@@ -247,15 +248,15 @@ internal val ObjCContainer.classOrProtocol: ObjCClassOrProtocol
  */
 internal fun Type.isStret(target: KonanTarget): Boolean {
     val unwrappedType = this.unwrapTypedefs()
-    return when (target) {
-        KonanTarget.IOS_ARM64 ->
+    return when (target.architecture) {
```

Comment:
Checking only architecture here is wrong, as targets with same architecture can have different ABIs.

Reply:
+1, guess current approach with explicit selection seems better fitting.